src/index.ts

```diff
@@ -27,7 +26,7 @@ import {
 import { middleware as requestIDsAdder } from "./lib/requestIDs"
 import { nameOldEigenQueries } from "./lib/modifyOldEigenQueries"
 import { rateLimiter } from "./lib/rateLimiter"
-import { graphqlErrorHandler } from "./lib/graphqlErrorHandler"
+// import { graphqlErrorHandler } from "./lib/graphqlErrorHandler"
```

Comment:
@zephraph Hmm, are we going to miss runtime exceptions being recorded in Sentry?